TestProject/output/BaseOutput.php

```diff
@@ -0,0 +1,52 @@
+<?php
+namespace output;
+
+use options\Getopt;
+use cellularAutomat\Board;
+
+/**
+ * BaseOutput class which provides the functions for pluggable-output.
+ *
+ * @class BasOutput
+ */
+class BaseOutput
+{
+    /**
+     * @var int creates a new image*.png after every run.
+     */
+    protected $imageIndex = 0;
+
+    /**
+     * Provides additional options.
+     *
+     * @param Getopt $_options
+     */
+    function addOptions(Getopt $_options)
+    {
+    }
+
+    /**
+     * Calls a single file.
```

Comment:
What do you mean by this? Call? File?
The reason that this method exists is so that outputs can initialize stuff and checked passed options so that they can create the correct output.